### PR TITLE
Method updates

### PR DESCRIPTION
**Pull request by pilotviper134**
- 'UIImagePNGRepresentation' has been replaced by instance method 'UIImage.pngData()'
- 'UIImageJPEGRepresentation' has been replaced by instance method 'UIImage.jpegData(compressionQuality:)'

### CODE REVIEW

**Comment by CaiJingLong** [comment]
Sorry, I will handle it tomorrow.

File: ios/Classes/UIImageHandler.swift [hunk 1]
```diff
@@ -33,9 +33,9 @@ class UIImageHandler {
 
     func outputMemory(format: FormatOption) -> Data? {
         if format.format == 0 {
-            return UIImagePNGRepresentation(image)
+            return UIImage.pngData(image)
```
Comment (CaiJingLong) [inline ios/Classes/UIImageHandler.swift]: The code shoule be :
`return image.pngData()`

File: ios/Classes/UIImageHandler.swift [hunk 2]
```diff
@@ -33,9 +33,9 @@ class UIImageHandler {
...
         } else {
-            return UIImageJPEGRepresentation(image, CGFloat(format.quality) / 100)
+            return UIImage.jpegData(image, CGFloat(format.quality) / 100)
```
Comment (CaiJingLong) [inline ios/Classes/UIImageHandler.swift]: Need replace to:
`return image.jpegData(compressionQuality: CGFloat(format.quality) / 100)`

**Comment by pilotviper134** [comment]
Changes looks good

**Pull request closed without merging**